tests/Hilke.EFloatCalculator.Tests/EFloatAlgebraicNumberCalculatorTests.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
```

Comment:
The folder is wrong, it should be `Hilke.KineticConvolution.EFloatCalculator.Tests`